core/src/main/java/site/ycsb/workloads/CoreWorkload.java

```diff
@@ -438,6 +453,28 @@ public void init(Properties p) throws WorkloadException {
     if (recordcount == 0) {
       recordcount = Integer.MAX_VALUE;
     }
+
+    String warmUpOpsProp = p.getProperty(WARM_UP_OPERATIONS_COUNT_PROPERTY, DEFAULT_WARMUP_OPS);
+    try {
+      warmUpOps = Long.parseLong(warmUpOpsProp);
```

Comment:
Could you introduce some small wrapper like parseInt(Properties p, String name, int defaultValue) and parseLong(Properties p, String name, long defaultValue) to avoid using such wrappers around.

Reply:
It's a philosophy question.
It depends on how much we want to change the original YCSB. I would prefer to keep it in a way the original YCSB have currently and do refactoring in a different ticket.